extension/adjustProductsForMapPricing.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function (context, input, cb) {
+  for (let product of input.products) {
+    if (product.flags.isMapProduct) {
```

Comment:
You also has to set the `msrp` price to null

Reply:
added line to set msrp line to null when isMapProduct flag is set to true.